pkg/manager/manager.go

```diff
@@ -209,6 +213,50 @@ func (m *SupportBundleManager) phaseCollectClusterBundle() error {
 	return nil
 }
 
+func (m *SupportBundleManager) phaseCollectPrometheusBundle() error {
+	pods, err := m.k8s.GetPodsListByLabels("cattle-monitoring-system", "app.kubernetes.io/name=prometheus")
```

Comment:
@c3y1huang Any objection to this feature? this feature adds a phase and checks as if the cluster has a Prometheus pod (especially run in the cattle-monitoring-system ns). If yes, it will try to extract the current alerts.

Reply:
No objections since it's non-blocking (optionalPhase). Additionally, Longhorn could potentially benefit from this.